Frontend/src/pages/ProfileCompletion.jsx

```
@@ -0,0 +1,188 @@
+        </div>
+      )}
+
+      {step === 2 && (
+        <div className="profile-step">
+          <h3 className="step-title">Step 2: Upload Profile Picture (Optional)</h3>
+          <label className="upload-box">
+            {formData.profilePicture ? <img src={formData.imagePreview} alt="Preview" className="profile-preview" /> : "Drag or Drop Your Profile Picture"}
+            <input type="file" name="profilePicture" onChange={handleFileChange} className="hidden" />
+          </label>
+          <div className="button-group">
+            <button onClick={() => setStep(1)} className="btn-secondary">Back</button>
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)